rules/game_process.tex

```diff
@@ -75,10 +81,20 @@ \subsection{Robot States}
   \label{fig:robot_states}
 \end{figure}
 
-The referee will announce the transition from the \texttt{initial} to \texttt{ready} state by raising both hands over its head. The referee will wear red gloves \footnote{Teams are encouraged to not rely on the red gloves as they could be removed in a following year.}.
-The referee must stand on the opposite side of the technical area behind the touchline and at the height of the halfway line.
+At the scheduled time, the referee will announce that the transition from the \texttt{initial} to \texttt{visual} will be executed 60 seconds later.
+The referee shall call ``Start in 60 seconds''
+The GameController operator shall start the countdown through the GameController.
+The GameController operator will announce the transition from \texttt{initial} to \texttt{visual} when the head referee is calling ``Start''.
+If all humans from playing teams have left the field, the head referee can announce the transition earlier.
+
+
+Once the game controller operator has confirmed the \texttt{visual} state, the referee should wait a random time between 10 to 120 seconds before announcing the transition to \texttt{ready}. 
+Teams should expect that the delay before the referee signals the transition from \texttt{visual} to \texttt{ready} will be different every time.
+The referee will announce the transition from \texttt{visual} to \texttt{ready} state by raising both hands over their head.
+The referee will wear red gloves \footnote{Teams are encouraged to not rely on the red gloves as they could be removed in a following year.}.
+The referee must stand on the opposite side of the technical area behind the touchline and in line with the halfway line.
```

Comment:
Three sentences beginning with "the". Maybe better:

To announce the transition from \texttt{visual} to \texttt{ready} state, the referee will raise both hands over their head.
Red gloves will be worn by the referee, though teams are encouraged not to rely on this visual cue as the gloves could be removed in a following year. 
The referee must stand on the opposite side of the technical area, behind the touchline, and in line with the halfway line.